### DISCUSSION TITLE
GW2 Radial will not work, help

### DISCUSSION THREAD

**Discussion by Tergazzi** [Friendly0Fire/GW2Radial#364] [Q&A]
Hello all, I had GW2 Radial working previously but upgraded my laptop and did a fresh install of windows, steam, GW2, etc. However, for some reason, I can not get GW2 Radial to work now. I have installed it via GW2 Addon Manager and manually. When in game if i hit "Shift+Alt+M" to bring up the menu/settings for it nothing happens. One thing I did notice that I thought was kinda odd, any addon i install through the GW2 Addon Manager doesnt seem to work. Which is ok since I only use ArcDPS and GW2Radial both which can be installed manually. I did ArcDPS manually and it works in game just fine.

Can someone please help me?

**Comment by Friendly0Fire** [comment]
Please use the issues system (with the correct template) in the future.

Since you say laptop, I am going to assume you are using an Intel CPU? Because if so, the addon loader is incompatible with Intel's drivers (which are active regardless of what discrete GPU you may have).

**Reply by Tergazzi** [reply]
So anyone that has Intel CPU is pretty much just boned? or is there a work around?

**Reply by Friendly0Fire** [reply]
On a laptop, unfortunately, yes. I have put some effort into a workaround but I am currently waiting for another dev to finish their part of the puzzle.

Desktop Intel CPUs are not affected. Desktop Intel GPUs are, but that's like five people so it's less urgent.

**Reply by Tergazzi** [reply]
Oh man, that would be so awesome when it is complete! I really dont want to use BlishHUD's Mounts & More addon, I find it visually unapealing...  lol
